### ISSUE TITLE
Title Tag & Meta Description Changes

### ISSUE THREAD

**Issue by Govej** [ONSdigital/sdg-indicators#3644]
Add functionality to edit title tags and meta descriptions

While not having a direct effect on how Google ranks the pages on the SDG data site, being able to edit and tailor the meta descriptions for specific pages may increase our click through rate and increase the organic traffic coming to the site

`<head>
<meta name="description" content="This is an example of a
meta description. This will often show up in search results.">
</head>`

Most of the title tags and meta descriptions for the site as default aren't awful but could definitely be optimized for the page they're linking to

**Comment by brockfanning** [comment]
@Govej To get us started here could you suggest a particular page and what its title and/or description should be?

**Comment by Govej** [comment]
Of course, when I'm back, I'll draft a list of H1s to change and title/meta description changes and send them off to the team for approval before I post them here
